packages/next/src/index.js

```diff
@@ -1,8 +1,14 @@
-const Prefresh = require('./plugin');
+const Prefresh = require('@prefresh/webpack');
 
 module.exports = (nextConfig = {}) => {
 	return Object.assign({}, nextConfig, {
 		webpack(config, options) {
+			if (!Prefresh.supportsNextJs) {
```

Comment:
Not sure whether we want such a check or not. I though it might be safer to have it instead of only relying on the dependency version. That could always be overridden by a resolve or some other shenanigans

Reply:
Yes this looks pretty legit to me